benchexec/tools/yogar-cbmc.py

```diff
@@ -28,24 +27,17 @@ class Tool(benchexec.tools.template.BaseTool):
     def executable(self):
         return util.find_executable('yogar-cbmc')
 
-    def version(self, executable):
-        return '0.1'
-
     def name(self):
-        return 'yogar-cbmc'
+        return 'Yogar-CBMC'
 
     def cmdline(self, executable, options, tasks, propertyfile, rlimits):
         options += ['--no-unwinding-assertions']
```

Comment:
Does your tool really always need this option? Or would there be cases where people might want to execute your tool without this option? Then please also remove it.